BP/common/decisions/10_religious_decisions.txt

```diff
@@ -0,0 +1,1083 @@
+﻿take_vow_of_poverty_decision = {
```

Comment:
In the commits it is indicated that this is being added to debug Jomini, is that all or is there anything else here? Because I'm not convinced of the utility of adding files to debug vanilla, that should rather be a bug report.

Reply:
Sorry that commit message was a bit ambiguous; I had to suppress a vanilla flag (cannot_be_atoned_krstjani_leader_flag) that was throwing Jomini errors. I've highlighted the exact changes in the file.